### DISCUSSION TITLE
Project Continuation

### DISCUSSION THREAD

**Discussion by FennyWilriani** [lab-brussels-1/HYF-Redesign-Group-3#57] [Ideas]
- Mision & Vision placed on the main page (to be finished by Zerubabel)
- Success story placed on the main page (to be finished by Josee)
- Readme.md/planning adjustment (to be finished by Fenny)
- Decide on the general rule:
   - [ ] font type
   - [ ] font size
   - [ ] color
   - [ ] naming convention: camelCase, dash, etc.
   - [x] use HTML & CSS only
   - [x] make it responsive 
- Which next page to work on: ***About Us & FAQ***
- FAQ example: [accordion responsive](https://www.youtube.com/watch?v=MXrtXg1kpVs&t=857s)
- Design (TBD)
- Divide task
   - About us: _History_ (@Ufitamahoro ), _Our Team_ (@andriivam ), _Title & icons_ (@zerubabel4 )
   - FAQ: @MMikhailova , Fenny (promise if next issue arise, it'll my sole responsibility to fix it)
   - @Melati5 , please inspect your code (contact us) and make it responsive, as it currently doesn't look all right on mobile.

**Comment by MMikhailova** [comment]
For the "About" page you need also assign Navbar to someone since it won't work without bootstrap links. We agreed to use only css&html.

Samir left comments in the Review discussion about naming in home page code.

Tamer explain about issues for navbar in discord.
